app/src/main/java/org/greenstand/android/TreeTracker/view/Tutorial.kt

```diff
@@ -0,0 +1,249 @@
+                DepthButton(
+                    modifier = Modifier
+                        .align(Alignment.Center)
+                        .size(width = 100.dp, 60.dp),
+                    isEnabled = false,
+                    onClick = { }
+                ) {
+                    Text(stringResource(R.string.note))
+                }
+                Image(
+                    modifier = Modifier.padding(top = 30.dp),
+                    painter = painterResource(id = R.drawable.touch_gesture),
+                    contentDescription = null
+                )
+            }
+
+        }
+        Spacer(modifier = Modifier.height(20.dp))
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight(),
+            horizontalArrangement = Arrangement.Start,
+            verticalAlignment = CenterVertically
+        ) {
+            Text(
+                text = stringResource(R.string.capture_tutorial),
+                color = CustomTheme.textColors.primaryText,
+                modifier = Modifier
+                    .fillMaxWidth(0.7f)
+                    .padding(end = 30.dp),
+                style = CustomTheme.typography.medium,
+                fontWeight = FontWeight.Bold,
+            )
+            Box(
+                modifier = Modifier
+                    .wrapContentHeight(),
+                contentAlignment = Center
+            ) {
+                ApprovalButton(onClick = { /*TODO*/ }, approval = true)
+                Image(
+                    modifier = Modifier.padding(top = 30.dp),
+                    painter = painterResource(id = R.drawable.touch_gesture),
+                    contentDescription = null
+                )
+            }
+
+        }
+        Spacer(modifier = Modifier.height(20.dp))
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight(),
+            horizontalArrangement = Arrangement.Start,
+            verticalAlignment = CenterVertically
+        ) {
+            Text(
+                text = stringResource(R.string.recapture_tutorial),
+                color = CustomTheme.textColors.primaryText,
+                modifier = Modifier
+                    .fillMaxWidth(0.7f)
+                    .padding(end = 30.dp),
+                style = CustomTheme.typography.medium,
+                fontWeight = FontWeight.Bold,
+            )
+            Box(
+                modifier = Modifier
+                    .wrapContentHeight(),
+                contentAlignment = Center
+            ) {
+                ApprovalButton(onClick = { /*TODO*/ }, approval = false)
+                Image(
+                    modifier = Modifier.padding(top = 30.dp),
+                    painter = painterResource(id = R.drawable.touch_gesture),
+                    contentDescription = null
+                )
+            }
+        }
+
+    }
+}
+
+@Composable
+fun TutorialDialog(
```

Comment:
This is fine for now. But ideally you'd reuse the dialog you created before. 

If you wanted to update the dialog style with your current implementation, you'd have to remember to update it in two locations instead of one.

We should have our own version of an "AlertDialog" that we can reuse.